docs/tags.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Document tags in eScriptorium
+summary: About using tags on documents in eScriptorium.
+authors:
+    - Alix Chagué
+date: 2023-04-25
+---
+
+# Tags on documents
+
+Tags are a useful feature to manage a project and its documents: they allow you to index your documents according to different criteria such as the state of completeness of the transcription task, the type of font, script or language used in them, or any other information you may require. Indeed, when you apply a tag to a document in a project, it becomes possible to filter the documents displayed in the dashboard according to the assigned tags. The following section describes how to use and manage the tags inside your project.
+
+Since you are free to set the labels you want, it is difficult to illustrate all the possible usage of such feature. Here are some examples:
+
+- tagging the script or language used in a document when the project combines several languages, it allows you to filter by language or script;
+- tagging the state of a document (for example "to segment", "segmented", "to transcribe", "to review", etc.);
+- tagging the team or people in charge of annotating/reviewing a document, it allows them to easily retrieve the document they have to work on;
+- tagging a bugged document to later resolution;
```

Comment:
I don't think I understand that phrase; should it be "to later resolve it" or "for a later resolution" ? Or maybe something else ?